agents/base/AgentItem.java

```diff
@@ -0,0 +1,6 @@
+package grakn.simulation.agents.base;
+
+public interface AgentItem {
+    String getTracker();
+    String getSessionKey();
+}
```

Comment:
This interface feels like a very "weak" abstraction. Why is it needed? What domain concept does it really represent? Do these 2 methods need to be enforced at all times? Could there also be a better name than `*Item`?

Reply:
Is it possible that this abstraction represents a `Resource` ?